javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/javaparsermodel/TypeExtractor.java

```diff
@@ -453,7 +453,7 @@ public ResolvedType visit(UnaryExpr node, Boolean solveLambdas) {
     @Override
     public ResolvedType visit(VariableDeclarationExpr node, Boolean solveLambdas) {
         if (node.getVariables().size() != 1) {
-            throw new UnsupportedOperationException();
+            throw new UnsupportedOperationException("The number of variables in VariableDeclarationExpr node is not 1");
```

Comment:
??? This message is visible to the user. It should be clearer.

Reply:
A clearer message would be for example "TypeExtractor support only one variable on a variable declaration expression".